views/base.njk

```diff
@@ -11,6 +11,7 @@
 {%- from 'medical-consent.njk' import medicalConsent as medicalConsent with context -%}
 {%- from 'patientName.njk' import patientName as patientName with context -%}
 {%- from 'condition-form.njk' import conditionForm as conditionForm with context -%}
+{%- from 'medication-form.njk' import medicationForm with context -%}
```

Comment:
can this not be added just to the medication.njk file?

Reply:
Decided to just group all of our custom templates here ... seems more organized